tests/unit/cmd/test_status.py

```diff
@@ -36,6 +36,7 @@ def test_api_status(cli_fs_runner, api_status_json_schema):
             All(
                 {
                     "detail": "Valid API key.",
+                    "instance": Match(r"https://[^\s]+\.com"),
```

Comment:
I'm not sure we want to match the `.com` here as it will fail when running the tests with a non default instance.
Maybe it's fine since other tests would also fail ? WDYT ?